src/bin/cosmogony.rs

```diff
@@ -24,6 +24,11 @@ struct Args {
     /// output file name
     #[structopt(short = "o", long = "output", default_value = "cosmogony.json")]
     output: String,
+    /// export as geojson file
+    #[structopt(long = "export-geojson", default_value = "true")]
```

Comment:
why not use `output` (as an `Option`)?

Reply:
It would change the current behaviour, would you mind?